classes/book.rb

```diff
@@ -0,0 +1,15 @@
+# The Book class represents a book with its title, author, and a list of rentals.
+class Book
+  # Getter and Setter for the title, author, and rentals of the book.
+  attr_accessor :title, :author, :rentals
+
+  # Initializes a new instance of the Book class with the given title and author.
+  # Params:
+  # +title+:: A string representing the title of the book.
+  # +author+:: A string representing the author of the book.
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = [] # Initialize an empty array to store the rentals associated with this book.
+  end
+end
```

Comment:
- [ ] Kindly add a method of adding a rental in book class to establish the `has many` association

```ruby
  def add_rental(person, date)
    Rental.new(date, self, person)
  end

```